### ISSUE TITLE
How to reduce fringe interference in results?

### ISSUE THREAD

**Issue by youyuge34** [xinntao/ESRGAN#31]
![](https://github.com/xinntao/ESRGAN/raw/master/figures/qualitative_cmp_02.jpg)

I got some fringe interference (条纹干涉) in results, just like `SRGAN results` in the above picture. And I don't know how to call about such phenomenon officially which is often happened in GAN training. 
It is worse in visual perception, and is there any way to reduce it? Or, which part of optimizations in ESRGAN reduce such phenomenon?       
THX!

**Comment by youyuge34** [comment]
And I have removed BN layers， which may not be the BN artifact

**Comment by xinntao** [comment]
@youyuge34 
Yes, this pattern of artifacts does not rise from BN layers. I cannot tell which specific part should take responsibility for these artifacts. I think better training will solve it. 
Maybe you can try the usage of VGG and RaGAN in ESRGAN.
